### ISSUE TITLE
"Duplicate" copies over incorrect data

### ISSUE THREAD

**Issue by bakoontz2** [a85/POSTMan-Chrome-Extension#11990]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Using "duplicate" on a collection item generates a duplicate with incorrect data:

![image](https://user-images.githubusercontent.com/38893590/236572679-c8f40abb-d2b5-4d3e-8b9b-12bd264e5fa2.png)

### Steps To Reproduce

1. Select a collection item using the 3 dots
2. Select Duplicate...
3. View the data.  It will not match the original collection item.

### Operating System

Windows 10 Enteprise

### Postman Version

10.13 (April 2023)

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by giridharvc7** [comment]
@bakoontz2 Can you ensure the changes are saved in the request you are trying to duplicate?

**Comment by bakoontz2** [comment]
Yes, they are saved.  In fact, I can replicate this by shutting postman down, then restarting and verifying requests are restored correctly.  Attempting to duplicate brings up random data.  I really can't discern a pattern, I don't know if the data is old cached data or what without further testing.  It does appear to be cached data...is there some way to disable this?
